### PR TITLE
Remove sensitive information from models and storage

### PR DESCRIPTION
**Pull request by beervoley**
In this PR I addressed [ACB33917](https://microsoftgarage.visualstudio.com/Intern%20GitHub/_workitems/edit/33917) and removed HCI from our models and database.
We previously stored first name, last name, team and organization of each user. We don't do it anymore. The only piece of information that's HCI classified is oid (authId in user model). This will be addressed in future PR.

### CODE REVIEW

**Comment by kodyang** [comment]
Do any frontend changes need to be made?

**Comment by beervoley** [comment]
> Do any frontend changes need to be made?

Not at the moment, no

File: private-templates-service/adaptivecards-templating-client/swagger.yaml [hunk 1]
```diff
@@ -517,18 +505,8 @@ definitions:
...
-        type: array
-        items: 
-          type: string
-      org: 
-        type: array
-        items: 
-          type: string
   TemplateInfo: 
     type: object
     properties:
```
Comment (gracez72) [inline private-templates-service/adaptivecards-templating-client/swagger.yaml]: Can remove the PostedUser definition and post user endpoint as well!

Comment (kodyang) [reply private-templates-service/adaptivecards-templating-client/swagger.yaml]: has this been resolved?

**Pull request merged**